mailbox/postgres/src/test/java/org/apache/james/mailbox/postgres/mail/PostgresModSeqProviderTest.java

```diff
@@ -0,0 +1,105 @@
+/****************************************************************
```

Comment:
I can't find the test class for JPAModSeqProvider, 
The `PostgresModSeqProviderTest` is based on Cassandra test cases.
Some with PostgresUidProvider

Reply:
I think it's good you bassed it on cassandra tests, as JPA does not have separate tables for uid and modseq. So Cassandra is the closest example then